src/lib.rs

```diff
@@ -86,7 +86,7 @@
     unused_unsafe,
     unused_variables,
     variant_size_differences,
-    warnings,
+//    warnings,
```

Comment:
Is this intentional, or was it supposed to be removed before you submitted the PR?